packages/designsystem/src/helpers/transformText.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Allow to create a slug in snake_case from a string
+ */
+export const createSlug = (value: string | null) => {
+  if (!value) return ''
+
+  // https://stackoverflow.com/a/37511463
+  return value
+    .toLocaleLowerCase() // Convert to lower case
+    .normalize('NFD') // Return Unicode Normalization Form
+    .replace(/[\u0300-\u036F]/g, '') // Remove accent
+    .replace(/\s{2,}/, ' ') // Remove multiple space to one
+    .replace(/ /g, '_') // Replace space by underscore
+    .replace(/[^a-z0-9_]/g, '') // For prevention
+}
+
+export default {
```

Comment:
I think you can omit to export a default in a module way.

Reply:
ok, i will try